onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2723,10 +2723,14 @@ static constexpr OrtApi ort_api_1_to_18 = {
     &OrtApis::SetDeterministicCompute,
     &OrtApis::KernelContext_ParallelFor,
     &OrtApis::SessionOptionsAppendExecutionProvider_OpenVINO_V2,
+    // End of Version 17 - DO NOT MODIFY ABOVE (see above text for more information)
+
     &OrtApis::SessionOptionsAppendExecutionProvider_VitisAI,
     &OrtApis::KernelContext_GetScratchBuffer,
     &OrtApis::KernelInfoGetAllocator,
-    &OrtApis::AddExternalInitializersFromFilesInMemory};
+    &OrtApis::AddExternalInitializersFromFilesInMemory
```

Comment:
nit: so future additions don't need to update this line

```suggestion
    &OrtApis::AddExternalInitializersFromFilesInMemory,
```